cucumber/protractor-chrome.config.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const GLOBAL_TIMEOUT = 40e3;
+
+exports.config = {
+  specs: "features/**/*.feature",
+  capabilities: {
+    browserName: "chrome",
+  },
+  directConnect: true,
+  cucumberOpts: {
+    require: ["./step_definitions/**/*.js"],
+    tags: ["~@wip"],
+    format: ["progress", "json:cucumber.json"],
```

Comment:
Trick: If you exchange this json reporter line  to the following line: "format-options": "{\"colorsEnabled\":true}",
you will get colorfull logs in command line.

Reply:
checked it out